dist/init/freebsd/caddy

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+#
+
+# PROVIDE: caddy
+# REQUIRE: LOGIN NETWORKING named cleanvar sshd
+# KEYWORD: shutdown
+
+#
+# Add the following lines to /etc/rc.conf to enable caddy:
+# caddy_enable (bool):        Set to "NO" by default.
+#                             Set it to "YES" to enable caddy
+#
+# caddy_cert_email (str):     Set to "" by default.
+#                             Defines the SSL certificate issuer email. By providing an
+#                             email address you automatically agree to letsencrypt.org's
+#                             general terms and conditions
+#
+# caddy_bin_path (str):       Set to "/usr/local/bin/caddy" by default.
+#                             Provides the path to the caddy server executable
+#
+# caddy_cpu (str):            Set to "99%" by default.
+#                             Configures, how much CPU capacity caddy may gain
+#
+# caddy_config_path (str):    Set to "/usr/local/www/Caddyfile" by default.
+#                             Defines the path for the configuration file caddy will load on boot
+#
+# caddy_run_user (str):       Set to "root" by default.
+#                             Defines the user that caddy will run on
+#
+
+. /etc/rc.subr
+
+name="caddy"
+rcvar=${name}_enable
+
+load_rc_config $name
+: ${caddy_enable:=no}
+: ${caddy_cert_email=""}
+: ${caddy_bin_path="/usr/local/bin/caddy"}
+: ${caddy_cpu="99%"} # was a bug for me that caused a crash within jails
```

Comment:
Interesting, thanks for the info!

So the 99%/100% bug, I wonder if that's a bug in Go or FreeBSD or Caddy? All we do is call `runtime.GOMAXPROCS(n)`. That sounds like an awful bug, whatever it is...